src/style.css

```diff
@@ -1,3 +1,108 @@
-body {
+html {
+  --color: #00acdc;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+header {
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  background-color: #fff;
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 60px;
+  text-transform: uppercase;
```

Comment:
Avoid fixing container sizes such as `width: 100%;` on the `header` unless there is a specific requirement for it. Let the content define the size to prevent overflow or accidental scroll bars.